src/NHibernate.DomainModel/NHibernate.DomainModel.csproj

```diff
@@ -12,6 +12,11 @@
     <NoWarn>$(NoWarn);3001;3002;3003;3005</NoWarn>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
```

Comment:
I would like to give it a try with only the release build checking that. If that generates frequently PR failing at this but detected a bit too late (especially already merged), it would be easy and fast to change that.